src/api-docs/swagger.ts

```diff
@@ -7,8 +7,13 @@ const doc = {
     version: '1.0.0',
     description: 'API documentation for the OrganizeMe project',
   },
-  host: 'organizeme-xtcl.onrender.com',
-  schemes: ["https"],
+  // Added to allow faster switching from prod to test, comment renders and uncomment test to switch
+
+  host: 'localhost:3000',
```

Comment:
This should not be localhost which will then update the output json

Reply:
removed